be1-go/hub/standard_hub/hub_state/peers.go

```diff
@@ -34,17 +36,17 @@ func (p *Peers) AddPeerInfo(socketId string, info method.ServerInfo) {
 func (p *Peers) AddPeerGreeted(socketId string) {
 	p.Lock()
 	defer p.Unlock()
-	if slices.Contains(p.peersGreeted, socketId) {
+	if slices.Contains(maps.Keys(p.peersGreeted), socketId) {
 		return
 	}
```

Comment:
```suggestion
	if _, ok := p.peersGreeted[socketId]; ok {
		return
	}
```

This could be implemented in a much faster way like this.
But do we even need it ?
The operation `p.peersGreeted[socketId] = struct{}{}` is idempotent, we might as well just go through it every time.

```suggestion
```